arbiter/server.go

```diff
@@ -266,7 +266,10 @@ func syncBinlogs(source <-chan *reader.Message, ld loader.Loader) {
 	dest := ld.Input()
 	for msg := range source {
 		log.Debug("recv msg from kafka reader", zap.Int64("ts", msg.Binlog.CommitTs), zap.Int64("offset", msg.Offset))
-		txn := loader.SlaveBinlogToTxn(msg.Binlog)
+		txn, err := loader.SlaveBinlogToTxn(msg.Binlog)
+		if err != nil {
```

Comment:
do not silently, if err is not nil, txn is not a valid value, think about how to handle it